eks-anywhere-common/Testers/Kong/test-job-configmap.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-configmap
+  namespace: kong
+data:
+  test.sh: |-
+    #!/bin/bash
+    kubectl apply -f https://bit.ly/echo-service
+    echo "
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
```

Comment:
The resources like Pod, svc, ingress which get created with test job is not getting cleaned up after the test job is completed. Please add a cleanup step to clean these resources. We dont want the resources to run ever.

Reply:
this was included, pls review